lib/json_schema_form/umm_preview_elements/umm_preview_contacts.rb

```diff
@@ -3,14 +3,15 @@
 # :nodoc:
 # TODO: MMT-2267 Abstract these and tool preview elements to minimize repeated
```

Comment:
Since it looks like this file works without significant changes, I think this can be removed as well.

Reply:
same as comment above